spec/services/migration/migrators/declaration_spec.rb

```diff
@@ -5,16 +5,17 @@
     let(:records_per_worker_divider) { 2 }
 
     def create_ecf_resource
-      create(:ecf_migration_participant_declaration, :ineligible)
+      travel_to(rand(100).hours.ago) do
+        create(:ecf_migration_participant_declaration, :ineligible)
+      end
     end
 
     def create_npq_resource(ecf_resource)
-      lead_provider = create(:lead_provider, ecf_id: ecf_resource.cpd_lead_provider.npq_lead_provider.id)
-      cohort = create(:cohort, ecf_id: ecf_resource.cohort_id, start_year: ecf_resource.cohort.start_year)
+      create(:lead_provider, ecf_id: ecf_resource.cpd_lead_provider.npq_lead_provider.id)
```

Comment:
could we revert those to what they were?

Reply:
ok will do.. but I thought without creating an existing declaration in NPQ it was a more real life scenario as we don't have any declarations in production.